classes/03class/exercises/c03-docker02/carineoliveira/Dockerfile.v2

```diff
@@ -0,0 +1,2 @@
+FROM httpd:2.4.41-alpine 
+EXPOSE 80
```

Comment:
the /usr/local/apache2/htdocs/ must be empty and the image being used already contain files there.

Reply:
thanks, added the command on the last commit.